### PR TITLE
(feat): add kafka-broker-pod-failure chart

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

-  Add kafka broker pod failure experiment chart - with list of permissions

**Logs of the job **
[logs-kafka-broker-pod-failure.txt](https://github.com/litmuschaos/chaos-charts/files/3839920/logs-kafka-broker-pod-failure.txt)

### CODE REVIEW

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  categoryDescription: |
+    kafka-broker-pod-failure contains chaos to disrupt state of kubernetes resources. 
+  keywords:
+    - Kubernetes
+    - Pod
+    - State 
+    - Kafka
+  platforms:
+    - GKE
+    - Convey
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: konvoy

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
+    vendor: CNCF
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://slack.kubernetes.io/
+spec:
+  displayName: kafka-broker-pod-failure
+  categoryDescription: |
+    kafka-broker-pod-failure contains chaos to disrupt state of kubernetes resources.
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: This chaos experiment kills (random or specified) kafka broker pods

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+apiVersion: litmuchaos.io/v1alpha1
+kind: ChartServiceVersion
+metadata:
+  name: kafka-broker-pod-failure
+  version: 0.1.0
+  annotations:
+    categories: Kubernetes
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: Kafka

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    vendor: CNCF
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: Empty

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    repository: https://github.com/litmuschaos/chaos-charts
+    support: https://slack.kubernetes.io/
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: our slack link in k8s

File: charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    - Convey
+  maturity: alpha
+  maintainers:
+    - name: ksatchit
+      email: karthik.s@mayadata.io
+  minKubeVersion: 1.12.0
+  provider:
+    name: Mayadata
+  links:
+    - name: Kubernetes Website
```
Comment (ksatchit) [inline charts/kafka/kafka-broker-pod-failure/kafka-broker-pod-failure.chartserviceversion.yaml]: apache kafka , confluent platform website, source code is litmuschaos/litmus

**Review by ksatchit — CHANGES_REQUESTED** [review]
minor changes

**Comment by ksatchit** [comment]
Lets also add the kafka (parent) chart-level csv and package yaml

**Review by ksatchit — APPROVED** [review]

**Review by chandankumar4 — APPROVED** [review]
LGTM

**Pull request merged**